internal/daemon/controller/listeners.go

```diff
@@ -276,13 +276,13 @@ func (c *Controller) configureForCluster(ln *base.ServerListener) (func(), error
 		}
 		go func() {
 			err := splitListener.Start()
-			if err != nil {
+			if !errors.Is(err, net.ErrClosed) {
```

Comment:
Should this still check for `nil` as well? I don't know if `Start` ever returns a `nil` error, but it might in the future?

Reply:
`errors.Is` won't match if it's nil, so I think it still serves that function...I might be missing something here.

There's nothing explicitly wrong with this message, but it throws an error in the log on normal shutdown...I noticed this while looking at the tests, and it seems nice to not error when expected. There is a slight change we could actually hide a real issue with the network listener closing but it seems unlikely that if there was an issue that we'd start the listener in the first place.

Reply:
> `errors.Is` won't match if it's nil

Right so, `errors.Is(nil, net.ErrClosed)` will be `false`, but we are doing `!errors.Is` so this `if` will be `true` when the error is `nil`. I don't think we want to write an error event in that case.

Reply:
Oh I see what you're saying. Gotcha. Yes, will change!

Reply:
Fixed